controller/api/apis.go

```diff
@@ -2597,28 +2597,43 @@ type RESTWorkloadInterceptData struct {
 	Intercept *RESTWorkloadIntercept `json:"intercept"`
 }
 
-type TagDetail struct {
-	ID          string `yaml:"id" json:"id"`
-	Title       string `yaml:"title" json:"title"`
-	Description string `yaml:"description" json:"description"`
+type RESTAvaiableComplianceFilter struct {
+	AvailableFilter []string `json:"available_filter"`
 }
 
 type RESTBenchCheck struct {
-	TestNum     string                   `json:"test_number"`
-	Category    string                   `json:"category"`
-	Type        string                   `json:"type"`
-	Profile     string                   `json:"profile"`
-	Scored      bool                     `json:"scored"`
-	Automated   bool                     `json:"automated"`
-	Description string                   `json:"description"`
-	Remediation string                   `json:"remediation"`
-	Tags        []map[string][]TagDetail `json:"tags,omitempty"`
+	TestNum     string                      `json:"test_number"`
+	Category    string                      `json:"category"`
+	Type        string                      `json:"type"`
+	Profile     string                      `json:"profile"`
+	Scored      bool                        `json:"scored"`
+	Automated   bool                        `json:"automated"`
+	Description string                      `json:"description"`
+	Remediation string                      `json:"remediation"`
+	Tags        []string                    `json:"tags,omitempty"`
+	TagsV2      map[string]share.TagDetails `json:"tags_v2,omitempty"` // TagsV2 provide compliance details for each compliance tag
+}
+
+type RESTProfileBenchCheck struct {
+	TestNum     string   `json:"test_number"`
+	Category    string   `json:"category"`
+	Type        string   `json:"type"`
+	Profile     string   `json:"profile"`
+	Scored      bool     `json:"scored"`
+	Automated   bool     `json:"automated"`
+	Description string   `json:"description"`
+	Remediation string   `json:"remediation"`
+	Tags        []string `json:"tags"`
 }
 
 type RESTBenchMeta struct {
 	RESTBenchCheck
 }
 
+type RESTProfileBenchMeta struct {
```

Comment:
where is this type referenced?

Reply:
I forget to delete this type, thanks for reminding.